src/main/java/com/google/sps/servlets/ImagesServlet.java

```diff
@@ -1,35 +1,84 @@
-package main.java.com.google.sps.servlets;
+package com.google.sps.servlets;
 
+//getting the images.java class...
```

Comment:
+1 on my comment on comments. If you can figure out what's going on by the import, you don't need to explain it in a comment.